### PR TITLE
Add guide for exposing vm as a service

### PR DESCRIPTION
**Pull request by duyanyan**
Draft the guide. Please kindly review. Thanks @fabiand @vladikr

### CODE REVIEW

File: vm-creation.md [hunk 1]
```diff
@@ -38,3 +38,84 @@ spec:
...
+  selector:
+    kubevirt.io: virt-launcher
+    kubevirt.io/domain: testvm-ephemeral
+  sessionAffinity: None
+  type: NodePort
+status:
+  loadBalancer: {}
+```
+
+You just need to create `NodePort service` by using `kubectl`:
```
Comment (rmohr) [inline vm-creation.md]: You just need to create **this**

File: vm-creation.md [hunk 2]
```diff
@@ -38,3 +38,84 @@ spec:
         lun: 2
         targetPortal: iscsi-demo-target.kube-system.svc.cluster.local
 ```
+
+### Expose VirtualMachine as a service
+
+Once the VirtualMachine is started, in order to connect to a VirtualMachine, you can create `Service` object for a VirtualMachine.
```
Comment (rmohr) [inline vm-creation.md]: you can create **a** `Service` object

File: vm-creation.md [hunk 3]
```diff
@@ -38,3 +38,84 @@ spec:
...
+```bash
+$ kubectl -f nodeport.yaml
+# OR
+$ kubectl get pod
+NAME                                   READY     STATUS    RESTARTS   AGE
+virt-launcher-testvm-ephemeral-9bqv4   2/2       Running   0          10m
+$ kubectl expose pod virt-launcher-testvm-ephemeral-mxjh8 --port=27017 --target-port=22 --type=NodePort --name=nodeportsvc
+```
+
+Service	can be listed by querying for the service objects:
```
Comment (rmohr) [inline vm-creation.md]: **The** Service can be listed

File: vm-creation.md [hunk 4]
```diff
@@ -38,3 +38,84 @@ spec:
...
+
+```bash
+$ kubectl get service
+NAME           TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)           AGE
+nodeportsvc    NodePort   172.30.232.73   <none>        27017:30000/TCP   5m
+```
+
+Connect the VirtualMachine by node IP and node port outside the clusternetwork:
```
Comment (rmohr) [inline vm-creation.md]: Connect **to** the VirtualMachine by **using a** node IP and the node port from outside of the cluster-network:

File: vm-creation.md (hunk 1 above)
Comment (duyanyan) [reply vm-creation.md]: updated

**Comment by duyanyan** [comment]
@rmohr Updated all the lines you mentioned. Thanks

**Comment by rmohr** [comment]
Maybe we should create an extra "service" menu item and not just attach it to the create-vm.md. The content look good.

**Comment by duyanyan** [comment]
@rmohr OK, moved it to a extra service menu item.

**Comment by rmohr** [comment]
I think you will need to link it to SUMMARY.md, so that it shows up in the menu on gitbook.

**Comment by duyanyan** [comment]
@rmohr Sorry for missing the menu. It is ok now. Thank you

**Comment by fabiand** [comment]
This is actually where #782 could play a role.

There are other ways to address the issue that a user needs to export a pod port, but in the range of options are

- #782
- #744
- #818

This does not impact the mergability of this PR.

**Pull request merged**